### DISCUSSION TITLE
31100Cafe

### DISCUSSION THREAD

**Discussion by CheckingChen** [MaLuns/hexo-theme-async#70] [Show 分享展示]
Name: 31100Cafe
Url: https://www.biuling.top/
Avatar: https://cdn.staticaly.com/gh/CheckingChen/image-hosting@master/avatar.6awsc8i38s80.webp
Desc: Creativity,起飞！
跟风一下，小小魔改也能上展示嘛（

**Comment by MaLuns** [comment]
已添加😏

**Reply by CheckingChen** [reply]
Thanks~🤤

**Reply by CheckingChen** [reply]
不好意思，因为折腾博客换了一个主题，可以撤下展示了😥

**Reply by MaLuns** [reply]
> 不好意思，因为折腾博客换了一个主题，可以撤下展示了😥

okok
